http-client/src/main/java/io/micronaut/http/client/netty/ssl/NettyClientSslBuilder.java

```diff
@@ -22,11 +22,7 @@
 import io.micronaut.http.HttpVersion;
 import io.micronaut.http.client.HttpVersionSelection;
 import io.micronaut.http.netty.NettyTlsUtils;
-import io.micronaut.http.ssl.AbstractClientSslConfiguration;
-import io.micronaut.http.ssl.ClientAuthentication;
-import io.micronaut.http.ssl.SslBuilder;
-import io.micronaut.http.ssl.SslConfiguration;
-import io.micronaut.http.ssl.SslConfigurationException;
+import io.micronaut.http.ssl.*;
```

Comment:
please configure your IDE not to replace with star imports

Reply:
@ChaimaaeROUAI done via https://github.com/micronaut-projects/micronaut-core/pull/11521/commits/7305619d3c6e213b92da7eae3fdef38d09dda120